pkgs/development/python-modules/nbconvert/default.nix

```diff
@@ -19,6 +19,7 @@
 , tornado
 , jupyter_client
 , defusedxml
+, substituteAll
```

Comment:
Don't think this is needed, because you use the substitute* bash functions, instead of the substitute* Nix function.

Reply:
Yes, indeed, I forgot it after I first tried to use `substituteAll` similarly as suggested in another review comment. I now removed this, thanks! :+1: